test/smoke_test/bcda_client.go

```diff
@@ -84,6 +85,26 @@ func main() {
 	}
 }
 
+func NewClient(accessToken string, retries int) *client {
+	c := &client{accessToken: accessToken}
+
+	retryClient := retryablehttp.NewClient()
+	retryClient.RetryMax = retries
+	retryClient.CheckRetry = func(ctx context.Context, resp *http.Response, err error) (bool, error) {
+		if resp.StatusCode == http.StatusUnauthorized {
+			log.Info("Access token expired. Refreshing...")
+			if err := c.updateAccessToken(); err != nil {
+				return false, fmt.Errorf("failed to update access token %s", err.Error())
+			}
+			return true, nil
+		}
+
+		return retryablehttp.DefaultRetryPolicy(ctx, resp, err)
+	}
+	c.httpClient = retryClient.StandardClient()
```

Comment:
Can we add the timeout values of 10s here? It was being done in the other HTTP client.

Reply:
Oh whoops, I forgot to add that back in, good catch!